pkg/e2e/operators/cloudingress/rhapi_lb.go

```diff
@@ -93,7 +119,137 @@ func testLBDeletion(h *helper.H) {
+					if err != nil {
+						fmt.Printf("GCP backend service already deleted. ")
+					} else {
+						_, err = computeService.BackendServices.Delete(project, oldLB.Name).Do()
+						if err != nil {
+							log.Printf("Error deleting backend service ")
+						}
+					}
+
+					ginkgo.By("Deleting GCP health check for rh-api ")
+					_, err = computeService.Addresses.Get(project, region, oldLB.Name).Do()
+					if err != nil {
+						fmt.Printf("GCP health check already deleted ")
+					} else {
+						_, err = computeService.ForwardingRules.Delete(project, region, oldLB.Name).Do()
+						if err != nil {
+							log.Printf("Error deleting health check ")
+						}
+					}
+
+					ginkgo.By("Deleting GCP target pool for rh-api ")
+					_, err = computeService.TargetPools.Get(project, region, oldLB.Name).Do()
+					if err != nil {
+						fmt.Printf("GCP target pool already deleted ")
+					} else {
+						_, err = computeService.ForwardingRules.Delete(project, region, oldLB.Name).Do()
+						if err != nil {
+							log.Printf("Error deleting target pool ")
+						}
+					}
+				}
+
+				ginkgo.By("Deleting GCP address for rh-api")
+				_, err = computeService.Addresses.Get(project, region, oldLBIP).Do()
+				if err != nil {
+					fmt.Printf("GCP IP address already deleted ")
+				} else {
+					_, err = computeService.Addresses.Delete(project, region, oldLBIP).Do()
+					if err != nil {
+						log.Printf("Error deleting address ")
+					}
+				}
+
+				newLBIP := ""
+				// Getting the new LB from GCP
+				err = wait.PollImmediate(15*time.Second, 5*time.Minute, func() (bool, error) {
+						// Getting the newly created IP from rh-api service
+						ginkgo.By("Getting new IP from rh-api service in OCM")
+
+						newLBIP, err = getLBForService(h, "openshift-kube-apiserver", "rh-api", "ip")
+						if (err != nil) || (newLBIP == "") || (newLBIP == oldLBIP) {
+							fmt.Printf("New rh-api svc not created yet...")
+							return false, nil
+						}else{
+							fmt.Printf("Found new rh-api svc! ")
+							fmt.Printf("new lb IP: %s ", newLBIP)
+							return true, nil
+						}
+					return false,nil
+				})
+				Expect(err).NotTo(HaveOccurred())
+
+				err = wait.PollImmediate(15*time.Second, 5*time.Minute, func() (bool, error) {
+					ginkgo.By("Polling GCP to get new forwarding rule for rh-api")
+					newLB, err := getGCPForwardingRuleForIP(computeService, newLBIP, project, region)
+					if err != nil || newLB == nil {
+						// Either we couldn't retrieve the LB, or it wasn't created yet
+						fmt.Printf("New forwarding rule not found yet...")
+						return false, nil
+					}
+					fmt.Printf("new lb name: %s ", newLB.Name)
+
+					if newLB.Name != oldLB.Name {
+						// A new LB was successfully recreated in GCP
+						return true, nil
+					}
+					// rh-api lb hasn't been deleted yet
+					fmt.Printf("Old forwarding rule not deleted yet...")
+					return false, nil
+				})
+				Expect(err).NotTo(HaveOccurred())
+			})
```

Comment:
Given that some of the codeblocks in the test are expected to take upwards of 5 minutes each, this might need an adustment of the expected timeout to avoid hitting the default timeout (which I think is 500 seconds currently)

ie something along the lines of
```
			},  10*time.Minute)
```
or thereabouts.

Reply:
Thanks for the change - though I was actually referring to the timeout for the `util.GinkgoIt` - I should have clarified better. By this, I mean - the code above has the structure:

```
util.GinkgoIt("Manually deleted LB should be recreated in GCP", func() {
})
```

There is an (optional) second argument, which is the expected timeout duration of the test. It will a default timeout of 500 seconds. But given the code above can take upwards of 600+ seconds, it will be wise to set that to 600.

So something like this:
```
util.GinkgoIt("Manually deleted LB should be recreated in GCP", func() {
}, 600)
```

Reply:
Yeah, I realized that later. I've added 600 for both. 

These tests don't seem to be running in the pipeline for some reason.